doc/nrf/links.txt

```diff
@@ -225,53 +225,53 @@
 
 .. ### Source: developer.nordicsemi.com
```

Comment:
Shouldn't all the "docs.nordicsemi.com" url be moved to ".. ### Source: docs.nordicsemi.com" ? Most of them are still uder developer.nordicsemi.com.

Reply:
May be the whole section be renamed as .. ### Source: docs.nordicsemi.com, all the developer URLs could be moved out to a separate section under ### Source: developer.nordicsemi.com

Reply:
We already have a section with ### Source: docs.nordicsemi.com. We could move these and other urls in this section to "### Source: docs.nordicsemi.com"